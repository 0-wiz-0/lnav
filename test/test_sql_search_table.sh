#! /bin/bash

export YES_COLOR=1

run_cap_test ${lnav_test} -n \
    -c ';SELECT * FROM procstate_procs' \
    ${test_dir}/logfile_procstate.0

run_cap_test ${lnav_test} -n \
    -c ';SELECT *,log_body FROM vpx_lro_begin' \
    ${test_dir}/logfile_vpxd.0
