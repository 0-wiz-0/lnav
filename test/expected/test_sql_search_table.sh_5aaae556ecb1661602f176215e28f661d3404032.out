[1mlog_line [0m[1mlog_part [0m[1m       log_time         [0m[1m[7mlog_idle_msecs [0m[1mlog_level [0m[1mlog_mark [0m[1mlog_comment [0m[1mlog_tags [0m[1mlog_filters [0m[1mmatch_index [0m[1muser [0m[1mpid [0m[1m[7mcpu_pct [0m[1m[7mmem_pct [0m[1m[7mvsz [0m[1m[7mrss [0m[1mtty [0m[1mstat [0m[1mstart_time [0m[1mcpu_time [0m[1m   cmd     [0m[1m cmd_name  [0m[1mcmd_args [0m
[1m       0[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2022-06-02 00:01:01.000[0m[1m [0m[1m             0[0m[1m [0m[1minfo     [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m          1[0m[1m [0m[1mroot[0m[1m [0m[1m  2[0m[1m [0m[1m    0.0[0m[1m [0m[1m    0.0[0m[1m [0m[1m  0[0m[1m [0m[1m  0[0m[1m [0m[1m?  [0m[1m [0m[1mS   [0m[1m [0m[1mJun01     [0m[1m [0m[1m0:00    [0m[1m [0m[1m[kthreadd][0m[1m [0m[1m[kthreadd][0m[1m [0m[1m         [0m
[7m      12[0m[7m [0m[7m  <[0m[7mNULL[0m[7m>[0m[7m [0m[7m2022-06-02 00:02:01.000[0m[7m [0m[7m         60000[0m[7m [0m[7minfo     [0m[7m [0m[7m       0[0m[7m [0m[7m     <[0m[7mNULL[0m[7m>[0m[7m [0m[7m  <[0m[7mNULL[0m[7m>[0m[7m [0m[7m     <[0m[7mNULL[0m[7m>[0m[7m [0m[7m          1[0m[7m [0m[7mroot[0m[7m [0m[7m  2[0m[7m [0m[7m    0.0[0m[7m [0m[7m    0.0[0m[7m [0m[7m  0[0m[7m [0m[7m  0[0m[7m [0m[7m?  [0m[7m [0m[7mS   [0m[7m [0m[7mJun01     [0m[7m [0m[7m0:00    [0m[7m [0m[7m[kthreadd][0m[7m [0m[7m[kthreadd][0m[7m [0m[7m    [0m     
[1m[7m      30[0m[1m[7m [0m[1m[7m  <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m2022-06-02 00:03:01.000[0m[1m[7m [0m[1m[7m         60000[0m[1m[7m [0m[1m[7minfo     [0m[1m[7m [0m[1m[7m       0[0m[1m[7m [0m[1m[7m     <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m  <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m     <[0m[1m[7mNULL[0m[1m[7m>[0m[1m[7m [0m[1m[7m          1[0m[1m[7m [0m[1m[7mroot[0m[1m[7m [0m[1m[7m  2[0m[1m[7m [0m[1m[7m    0.0[0m[1m[7m [0m[1m[7m    0.0[0m[1m[7m [0m[1m[7m  0[0m[1m[7m [0m[1m[7m  0[0m[1m[7m [0m[1m[7m?  [0m[1m[7m [0m[1m[7mS   [0m[1m[7m [0m[1m[7mJun01     [0m[1m[7m [0m[1m[7m0:00    [0m[1m[7m [0m[1m[7m[kthreadd][0m[1m[7m [0m[1m[7m[kthreadd][0m[1m[7m [0m[1m[7m    [0m[1m     [0m
